docs/CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ The following `tip` changes can be tested by building VictoriaMetrics components
 * FEATUTE: [vmalert](https://docs.victoriametrics.com/vmalert.html): display the error message received during unsuccessful config reload in vmalert's UI. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4076) for details.
 * FEATUTE: [vmalert](https://docs.victoriametrics.com/vmalert.html): allow disabling of `step` param attached to [instant queries](https://docs.victoriametrics.com/keyConcepts.html#instant-query). This might be useful for using vmalert with datasources that to not support this param, unlike VictoriaMetrics. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4573) for details.
 * FEATUTE: [vmalert](https://docs.victoriametrics.com/vmalert.html): support option for "balckholing" alerting notifications if `-notifier.blackhole` cmd-line flag is set. Enable this flag if you want vmalert to evaluate alerting rules without sending any notifications to external receivers (eg. alertmanager). See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4122) for details. Thanks to @venkatbvc for [the pull request](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4639).
+* FEATUTE: [vmalert](https://docs.victoriametrics.com/vmalert.html): add `keep_firing_for` field for alerting rule. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4529).
```

Comment:
I think we should have changes to:
* docs and rules config description
* to vmui for reflecting that alert is in Stabilizing state
* ensure that vmalert's APIs return the new field

Reply:
Sorry I missed the docs, updated.

About vmui:
In https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4669/commits/b3c686f7c069aa297fd87438f3c8111997e003fb, I added `keep_firing_for` info for alert rule if set and will show "Stabilizing" when firing state is kept because of `keep_firing_for` instead of real alert, does that look good?
<img width="1390" alt="image" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/39937150/136803be-dd02-43b3-8d57-0655e75521ac">
<img width="1392" alt="image" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/39937150/619f3e57-1c07-487a-b1be-16ef73cc71b8">